src/Microsoft.AspNet.Authentication/DataHandler/TicketSerializer.cs

```diff
@@ -36,98 +39,177 @@ public virtual AuthenticationTicket Deserialize(byte[] data)
             }
         }
 
-        public static void Write([NotNull] BinaryWriter writer, [NotNull] AuthenticationTicket model)
+        public virtual void Write([NotNull] BinaryWriter writer, [NotNull] AuthenticationTicket ticket)
```

Comment:
We can keep Write/Read public and make WriteIdentity/WriteClaim/ReadIdentity/ReadClaim protected if you want.